Blockzilla/AppDelegate.swift

```diff
@@ -66,11 +66,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 let shouldShowNewIntro = arc4random_uniform(UInt32(100)) >= 50
                 if  shouldShowNewIntro {
                     firstRunViewController = IntroViewController()
-                    Telemetry.default.recordEvent(category: TelemetryEventCategory.firstRun, method: TelemetryEventMethod.coinFlip, object: TelemetryEventObject.onboarding, value: nil, extras: ["DidShowNewIntro": shouldShowNewIntro])
```

Comment:
Why do you want to change this?

Reply:
Based on the pattern that Android is doing, this isn't as concise as it can be and doesn't need to be this verbose, unless we want it to be verbose

Reply:
But aren't you removing the result of the coin flip?

Reply:
The result of the coin flip is what is shown though, so why would we need it?

Reply:
That'a a fair point - we show onboarding or firstrun, cool.